### ISSUE TITLE
Integrate a site-wide search

### ISSUE THREAD

**Issue by hola-soy-milk** [distributeaid/distributeaid.org#349]
Perhaps using [Algolia](https://www.algolia.com/)

Goals:

- [ ] mention Algolia in the Data Privacy section of the page
- [ ] Integrate Algolia package
- [ ] Index site using Algolia
- [ ] Design/implement search bar
- [ ] Deploy hook to rebuild Algolia index

**Comment by coderbyheart** [comment]
We've used Algolia [with the COVID-19 resources project](https://github.com/distributeaid/covid-19-resources-drive-export/blob/d672758ebc71532eca63d6f6f1f80d6be7c7b55f/gatsby-config.js#L63-L134) and had good results.

**Comment by raae** [comment]
How much content is it? Could fuse.js work?

**Comment by coderbyheart** [comment]
Yes, absolutely. The site will not turn into a publishing platform, so having the search DB in the client will be fine.

**Comment by raae** [comment]
Here is my [fuse implementation](https://github.com/raae/queen.raae.codes/blob/main/src/pages/search/%5B...term%5D.js) on my own site if that is helpful.
